ARCs/arc-0034.md

```diff
@@ -0,0 +1,96 @@
+- Alignment of personal interests with the advancement of the Algorand ecosystem.
+> Issues requiring resolution may have been discussed on various online platforms such as forums, discord, and social media networks.
+
+### Voting System
+The voting system for the xGov program will be the following:
+- Only xGovs are eligible to participate in the voting process
+- Utilization of the Algo committed by xGov as a form of voting currency, with a 1:1 ratio of Algos to votes
+- Proposals that do not reach the threshold of votes required will be removed from consideration.
+
+### Threshold 
+In order for a proposal to be approved, it is necessary for the number of votes in favor of the proposal to be proportionate to the amount of funds requested. This ensures that the allocation of funds is in line with the community's consensus and in accordance with democratic principles.
+
+> eg. 2 000 000 Algo are available to be given away as grants, 200 000 000 millions Algo are committed to the xGov Process (200 000 000 votes available):
+> - Proposal A request 100 000 Algos (5 % of the Amount available)
+> - Proposal A needs 5% of the votes (10 000 000 Votes) to go through
+
+### Life of a proposal
+The proposal process will have the same duration as the governance period. Within the 3 months, the following steps will be implemented:
+- Anyone can submit a proposal at any time. 
+- 0.5 months after the start of a governance periode, proposals submitted enter the process
+- 1 month is allocated for proposals refinement and temperature check. Proposals will be evaluated and refined by the community and xGovs.
+- 1 month is allocated for voting on proposals. The community will vote on proposals that have passed the refinement and temperature check stage.
+- 0.5 months for negotiation and claiming grants. Proposals that have been approved will be negotiated and grants will be claimed.
+
+### Submit a proposal
+In order to submit a proposals, a proposer needs to create a pull request on the following repository: <a href="https://github.com/algorandfoundation/xGov">xGov Proposals</a>.
+
+Proposals **MUST**:
+- Follow the [template form provided](../assets/arc-0034/TemplateForm.md).
+- Follow the rules of the xGov Proposals Repository.
+- Have the status `Final` before the end of the temperature check. (1,5 Months after the start of a governance)
+
+### Voting on proposal
+At the start of the voting period (1,5 Months after the start of a governance) xGovs [ARC-33](arc-0033.md) will votes on proposals through an on-chain votes.
+
+Vote will refer to the PR number and a cid hash of the proposal itself. 
+
+The CID MUST:
+- Represent the file.
+
+- Be a version V1 CID
+  - E.g., use the option --cid-version=1 of ipfs add
+
+- Use SHA-256 hash algorithm
+  - E.g., use the option --hash=sha2-256 of ipfs add
+
+### Grants calculation
+The allocation of grants will be distributed taking into consideration the funding request amounts.
```

Comment:
The allocation of grants will consider the funding request amounts and the available amount of ALGO to be distributed. 
For example, Proposal N requests 100,000 ALGO in a pool of 2,000,000 ALGO, representing 5% of the pool. Therefore, in order for Proposal N to be approved, it will need to receive 5% of the total votes available. If there are 20,000,000 ALGO in the Term Pool(s), then Proposal N needs 1,000,000 votes.